draft-mouris-cfrg-mastic.md

```diff
@@ -205,11 +205,42 @@ into Mastic in {{vidpf}}. The appendix includes supplementary material:
+* `len(obj: Sized) -> int` returns the number of items in `obj`. The object argument can be a sequence (e.g., string, list, or tuple) or a collection (e.g., dictionary, set).
+* `list.append(elem: Any) -> None` adds a single element `elem` to the end of a `list` instance.
+* `list.copy() -> list` returns a shallow copy of a `list` instance.
+* `range(stop)` or `range(start, stop[, step])` is the range function from the Python standard library. The one-argument form returns the integers from zero (inclusive) to `stop`, exclusive. The two- and three-argument forms allow overriding the start of the range and overriding the step between successive output values.
+* `set(iterable: Optional[Iterable] = None) -> set` creates a new set object, which is an unordered collection of unique elements. The optional `iterable` argument (e.g., list, tuple, or string) is used to initialize the set. If no argument is provided, an empty set is created.
+* `to_le_bytes(val: int, length: int) -> bytes` converts `val` to little-endian bytes; its value MUST be in the range `[0, 2^(8*length))`.
+* `vec_add(left: list[F], right: list[F]) -> list[F]` adds the `right` list operand to the `left` list operand and returns the result.
+* `vec_neg(vec: list[F]) -> list[F]` negates the input vector `vec` and returns the result.
+* `vec_sub(left: list[F], right: list[F]) -> list[F]` subtracts the `right` list operand from the left list operand and returns the result.
+* `XofFixedKeyAes128(seed: bytes, dst: bytes, binder: bytes) -> Xof` is a XOF based on a circular collision-resistant hash function from fixed-key AES that is initialized with a given seed, a domain separation tag, and binder string, and returns a `Xof` instance. The length of `seed` will typically be SEED_SIZE, but some XOFs may support multiple sizes. The `seed` MUST be generated securely (i.e., it is either the output of a CSPRNG or a previous invocation of the XOF). The length of the domain separation string `dst` passed to XofTurboShake128 MUST NOT exceed 65535 bytes.
+* `XofTurboShake128(seed: bytes, dst: bytes, binder: bytes) -> Xof` is a XOF wrapper for TurboSHAKE128 that is initialized with a given seed, a domain separation tag, and binder string, and returns a `Xof` instance. The length of `seed` will typically be SEED_SIZE, but some XOFs may support multiple sizes. The `seed` MUST be generated securely (i.e., it is either the output of a CSPRNG or a previous invocation of the XOF). The length of the domain separation string `dst` passed to XofTurboShake128 MUST NOT exceed 65535 bytes.
+* `Xof.next(length: int) -> bytes` returns the next `length` bytes of output of a `Xof` instance.
+* `xor(left: bytes, right: bytes) -> bytes` returns the bitwise XOR of `left` and `right`. An exception is raised if the inputs are not the same length.
```

Comment:
likewise here: we don't need to list the `Xof` APi.

Reply:
We are listing all the functions we use so the description is self contained.